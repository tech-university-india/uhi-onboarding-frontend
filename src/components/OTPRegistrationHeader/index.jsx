import React from "react";
import TextBox from "../TextBox";

const OTPRegistrationHeader = () => {
  return (
    <>
      <TextBox placeholder="ABHA Number" disabled={true}/>
      <p className="m-6 text-center">
        Please enter ABHA number you want to use with ABHA
      </p>
      <TextBox placeholder="Mobile Number" />
    </>
  );
};

export default OTPRegistrationHeader;
