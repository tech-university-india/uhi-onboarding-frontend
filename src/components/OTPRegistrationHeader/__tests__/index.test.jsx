import { render, screen } from "@testing-library/react";
import OTPRegistrationHeader from "..";

describe("OTPRegistrationHeader", () => {
  it("should render correctly", () => {
    const { asFragment } = render(<OTPRegistrationHeader />);
    expect(asFragment()).toMatchSnapshot();
  });

  it("should render the correct text", () => {
    render(<OTPRegistrationHeader />);
    expect(screen.getByPlaceholderText("ABHA Number")).toBeInTheDocument();
    expect(screen.getByPlaceholderText("Mobile Number")).toBeInTheDocument();
  });

  it("should render the correct text", () => {
    render(<OTPRegistrationHeader />);
    expect(
      screen.getByText("Please enter ABHA number you want to use with ABHA")
    ).toBeInTheDocument();
  });
});
