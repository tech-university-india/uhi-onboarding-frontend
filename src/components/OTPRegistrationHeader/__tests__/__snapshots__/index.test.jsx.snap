// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OTPRegistrationHeader should render correctly 1`] = `
<DocumentFragment>
  <input
    class="disabled:text-preFillText p-3 w-full rounded-lg shadow-textBoxInset bg-textBox"
    disabled=""
    placeholder="ABHA Number"
    type="text"
    value=""
  />
  <p
    class="m-6 text-center"
  >
    Please enter ABHA number you want to use with ABHA
  </p>
  <input
    class="disabled:text-preFillText p-3 w-full rounded-lg shadow-textBox bg-textBox"
    placeholder="Mobile Number"
    type="text"
    value=""
  />
</DocumentFragment>
`;
